samples/openvx/openvx_interoperability.cpp

```diff
@@ -0,0 +1,324 @@
+#include <iostream>
+#include <stdexcept>
+
+//OpenVX includes
+#include <VX/vx.h>
+#include <VX/vx_lib_debug.h>
+#include <VX/vx_helper.h>
+#include <VX/vx_lib_extras.h>
+
+//for vx-Access/Commit-ImagePatch
+#include <VX/vx_compatibility.h>
```

Comment:
And maybe it will be better to completely avoid `vx_compatibility.h` usage, since it is a deprecated stuff.